### ISSUE TITLE
SEO Fixes - Images: Missing Alt Text

### ISSUE THREAD

**Issue by svikrantaxioned** [kiwix/web#210]
Link for the URLs - https://docs.google.com/spreadsheets/d/1adMahF8Ta5t1M9WE-j7d7ows5cKYr23DFw531yYAolc/edit#gid=1609106425

Priority
Medium

Description
Images that have an alt attribute, but are missing alt text. Click the address of the image and then the 'Image Details' tab in the lower window pane to view which pages have the image on them, and are missing alt text. Images should have descriptive alternative text about its purpose, which helps the blind and visually impaired, and the search engines understand it and its relevance to the web page.

Resolution
Include descriptive alt text for images to help users and the search engines understand them better. Where possible, decorative images should be provided using CSS background images or alternatively a null (empty) alt text should be provided (alt="") so that they can be ignored by assistive technologies, such as screen readers.
**Free Plugins are available to do bulk URL fixes**

**Comment by kelson42** [comment]
> Could not edit the alt-text for those two pictures that are actually hosted on diff, the Wikimedia blog.

Really surprised about that! I can hardly believe that WP is that bad! No way to just import these two pictures within our WP (woyld be more robust as well)?
